go-controller/pkg/ovn/ovn.go

```diff
@@ -339,7 +338,7 @@ func podScheduled(pod *kapi.Pod) bool {
 
 // WatchPods starts the watching of Pod resource and calls back the appropriate handler logic
 func (oc *Controller) WatchPods() error {
-	retryPods := sets.String{}
+	var retryPods sync.Map
```

Comment:
did you intend to leave this in given that you aren't changing pod event handling?

Reply:
@danwinship yeah, kinda; I still intend to change pod handling in the near future, but we need to rework how our network policy is handled to fix the race between pod addition (which would be faster because it's parallel) and network policy creation.